Dart/src/com/jetbrains/lang/dart/ide/errorTreeView/DefaultDartFeedbackBuilder.java

```diff
@@ -3,25 +3,51 @@
 import com.intellij.ide.BrowserUtil;
 import com.intellij.openapi.application.ex.ApplicationInfoEx;
 import com.intellij.openapi.project.Project;
+import com.intellij.openapi.util.io.FileUtil;
 import com.jetbrains.lang.dart.DartBundle;
 import com.jetbrains.lang.dart.sdk.DartSdk;
 import org.jetbrains.annotations.Nullable;
 
+import java.io.File;
+import java.io.IOException;
+
 import static com.intellij.ide.actions.SendFeedbackAction.getDescription;
 
 public class DefaultDartFeedbackBuilder implements DartFeedbackBuilder {
+  private String myMessage;
```

Comment:
Is it possible to make the class stateless and pass a message as a parameter of sendFeedback() method instead?